tests/jsunit/workspace_test.js

```diff
@@ -235,8 +235,8 @@ function test_renameVariable_ReferenceExists() {
 }
 
 function test_renameVariable_TwoVariablesSameType() {
-  // Expect 'renameVariable' to change oldName variable name to newName.
-  // Expect oldName block name to change to newName
+  // Cannot rename variable to a name that already exists
```

Comment:
For these tests, add a note to each one to say that it's different from the behaviour in blockly.  It'll help us in the future.

Reply:
Done.